scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -122,6 +154,22 @@ function standardizeDescription(desc: string): string {
   return desc;
 }
 
+function isPlausibleLowIncomeRow(record: Record<string, string>) {
+  if (
+    record.income_restrictions !== undefined &&
+    record.income_restrictions !== ''
+  ) {
+    return true;
+  }
+  if (
+    record.short_description !== undefined &&
+    record.short_description.toLowerCase().includes('income')
```

Comment:
curious if you've seen this included in all/most of the incentive descriptions with income requirements? Are there any other patterns?

Also worth double checking if we want to be more conservative for low income rows (i.e. if we only want to record them if we're 100% certain they're defined or not)

Reply:
This popped up a few times in CO – Maria flagged some cases where the short description referenced income eligibility, and Dena or I just hadn't populated the Income Restrictions for some reason. This check will flag those prior to human review.

it's just a warning, so the human can still just ignore them if the thresholds don't exist/shouldn't be defined, etc.